draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -323,6 +369,7 @@ private void mapGUI(TypedArray attributes) {
         attributes.getResourceId(R.styleable.draggable_view_bottom_view_id, R.id.second_view);
     dragView = findViewById(dragViewId);
     secondView = findViewById(secondViewId);
+
```

Comment:
dragViewId and secondViewId are local variable and it has no sense to declare it as final variables. dragView and secondView are not declared as final because that attributes are instantiated when the layout is inflated and you can't use final modifier with an attribute if you don't initialize in the constructor.